cmake/FindTarantool.cmake

```diff
@@ -23,7 +23,7 @@ if(TARANTOOL_INCLUDE_DIR)
 endif()
 
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(TARANTOOL
+find_package_handle_standard_args(Tarantool
```

Comment:
It seems, here we meet the same problem as one that was fixed in https://github.com/tarantool/modulekit/pull/6

Reply:
```sh
$ ag find.req
CMakeLists.txt
12:set(TARANTOOL_FIND_REQUIRED ON)
```

Reply:
Thanks! Fixed in a separate commit.